.github/workflows/build.yaml

```diff
@@ -0,0 +1,288 @@
+           -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+           -Dgalasa.release.repo=file:${{ github.workspace }}/obr/repo \
+           --batch-mode --errors --fail-at-end \
+           --settings  /home/runner/work/gpg/settings.xml
+        
+        - name:  Generate Galasa OBR
+          run: |
+           docker run --rm -v ${{github.workspace}}:/var/root/ ghcr.io/${{env.NAMESPACE}}/galasabld-amd64:${{env.IMAGE_TAG}} template --releaseMetadata var/root/framework/release.yaml --releaseMetadata /var/root/extensions/release.yaml --releaseMetadata /var/root/managers/release.yaml --releaseMetadata /var/root/obr/release.yaml --template /var/root/obr/dev.galasa.uber.obr/pom.template --output /var/root/obr/dev.galasa.uber.obr/pom.xml --obr
+            
+        - name: Display Galasa OBR pom.xml
+          run: |
+           cat ${{github.workspace}}/obr/dev.galasa.uber.obr/pom.xml
+         
+        - name: Build Galasa OBR with maven
+          run: |
+            mvn -f ${{github.workspace}}/obr/dev.galasa.uber.obr/pom.xml deploy \
+            -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/managers \
+            -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+            -Dgalasa.release.repo=file:${{ github.workspace }}/obr/repo \
+            --batch-mode --errors --fail-at-end \
+            --settings  /home/runner/work/gpg/settings.xml
+        
+        - name: Login to Github Container Registry
+          uses: docker/login-action@v3
+          with:
+            registry: ${{ env.REGISTRY }}
+            username: ${{ github.actor }}
+            password: ${{ secrets.GITHUB_TOKEN }}
+      
+        - name: Extract metadata for OBR image
+          id: metadata
+          uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+          with:
+            images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/obr-maven-artefacts
+    
+        - name: Build and Push OBR image 
+          id: build
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ${{github.workspace}}/obr/dockerfiles/dockerfile.obr
+            push: true
+            tags: ${{ steps.metadata.outputs.tags }}
+            labels: ${{ steps.metadata.outputs.labels }}
+            build-args: |
+                dockerRepository=${{env.REGISTRY}}
+                tag=${{ env.IMAGE_TAG }}
+        
+        - name: Recycle OBR application in ArgoCD
+          run: |
+            docker run --env ARGOCD_AUTH_TOKEN=${{ env.ARGOCD_AUTH_TOKEN }} --rm -v ${{ github.workspace }}:/var/workspace ghcr.io/galasa-dev/argocdcli:main app actions run gh-maven-repos restart --kind Deployment --resource-name obr-gh --server argocd.galasa.dev
+
+    
+  build-obr-javadocs:
+    name: Build OBR javadocs using galasabld image and maven
+    runs-on: ubuntu-latest
+    needs: build-obr

```

Comment:
This can be removed the jobs can run in parallel - please also make the same change in the PR workflow

Reply:
done